frontend/src/components/navigation/FourZeroFourDisplay.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    viewBox="0 0 1366 768"
+  >
+    <defs>
```

Comment:
```suggestion
    <title>Error 404</title>
    <defs>
```
Please make it accessible.